### PR TITLE
Add how to for requesting generic permission

### PR DESCRIPTION
**Pull request by Ahmad-Elsayed**
Generic permission should allow the users to request any available device permission. Will be merged to docs site when the nanoflow action is released in a few days.

Besides this PR, we will also update the following resources in a separate, future PR:
- [ ] Update Native Mobile Resources doc with a section for Nanoflow Actions (can simply name and describe each action)
- [ ] Update this GH page with an accurate list of actions https://github.com/mendix/widgets-resources

### CODE REVIEW

**Comment by Ahmad-Elsayed** [comment]
yet to be released

File: content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+{{< figure src="/attachments/howto/mobile/native-mobile/implementation/generic-permission-action/deployment-android" alt="deploy app with generic permission screenshoot"   width="400"  >}}
+{{< figure src="/attachments/howto/mobile/native-mobile/implementation/generic-permission-action/deployment-ios" alt="deploy app with generic permission screenshoot"   width="400"  >}}
+
+{{% alert color="info" %}}
+The permissions that you can't find using the Native builder UI, can be added manually referencing the following section.
+{{% /alert %}}
+
+## 5 Update native app permissions
+###5.1 iOS {#update-ios-native-permissions}
```
Comment (UrazAkgultan) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: You need to add empty space here.

```suggestion
### 5.1 iOS {#update-ios-native-permissions}
```

File: content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  	<!-- … -->
+	<key>com.apple.developer.siri</key>
+	<true/>
+</dict>
+</plist>
+
+```
+{{% /alert %}}
+
+###5.2 Android {#update-android-native-permissions}
```
Comment (UrazAkgultan) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: You need to add empty space here.

```suggestion
### 5.2 Android {#update-android-native-permissions}
```

**Review by UrazAkgultan — CHANGES_REQUESTED** [review]
Good work 👍 I've left two comments

File: content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md (hunk 1 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: done

File: content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md (hunk 2 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/generic-permission-action.md]: done

**Review by UrazAkgultan — APPROVED** [review]
LGTM 👍

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**